tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "noFallthroughCasesInSwitch": true,
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "downlevelIteration": true
```

Comment:
Can you explain what this change in `tsconfig.json` is for?

Reply:
I added `downlevelIteration` in order to support the use of the spread operator on a `Set`. In `InterestRatesView.tsx`, I used it like this: `const tokenAddresses = [...new Set(allTokens)];`

You can see more about `downlevelIteration` in the [tsconfig reference](https://www.typescriptlang.org/tsconfig#downlevelIteration).

Reply:
Awesome was trying to implement this exact thing yesterday. thanks